docs/rest_guidelines.md

```diff
@@ -121,8 +121,22 @@ The status codes are grouped in five classes:
 * 4xx client error – the request contains bad syntax or cannot be fulfilled
 * 5xx server error – the server failed to fulfill an apparently valid request
 
-Links to status code lists:
+In total there are over 60 possible HTTP Status codes, but APIs should focus on using a limited set in a consistent manner.
+
+Commonly used HTTP Status codes are: 
+* 200 OK - The request completed successfully
+* 201 Created - The request completed successfully and one or more new resources were created
+* 204 No Content - The request completed successfully and there is no additional content in the response (commonly used for HTTP PATCH)
```

Comment:
should the most used 30x be included? e.g. 301 and 304?

Reply:
Could be a good candidate. 

Attempted to find some more information on how common the usage of 301 and 302s are for APIs 
![image](https://user-images.githubusercontent.com/1133607/148730725-d42b8178-ffcd-4487-9135-851fe20c3bee.png)
Found a couple of experts (involved in writing some of these standards) recommending HTTP 307 for temporary redirects and 308 for permanent redirects.

The main reason for this is that if you reply HTTP 301/302 for a HTTP POST request, the browser/libraries follow up with a HTTP GET for the new url. Thereby, it switches the HTTP verb and any body payload is lost. 307/308 should not have that limitation.

So perhaps 307/308 are better candidates?

Reply:
Yeah, 307/308 seems to be more correct!

I think 304 is also quite often used if result is cached some time